bin/handlerFunctions.js

```diff
@@ -666,9 +666,14 @@ async function handleRawRequest(argv) {
 }
 
 // put file function
-async function pushFile(file, requestOptions, playerData, isRawData, isVerbose) {
+async function pushFile(file, destination, playerData, isRawData, isVerbose) {
     logIfOption(`Pushing file: ${file}`, isVerbose);
 
+    const requestOptions = {
+        method: 'PUT',
+        url: 'http://' + playerData[1] + '/api/v1/files/' + playerData[3] + '/' + destination,
```

Comment:
Minor point - It would be good to comment what is the structure of `playerData` expected to be received here? May be a function description would cover that.